### ISSUE TITLE
lint: Disabling multiple rules require additional newline

### ISSUE THREAD

**Issue by mrVanboy** [mgechev/revive#102]
**Describe the bug**
After commit https://github.com/mgechev/revive/commit/c878d3090df36b2b57f9427ec7ea29be528e19ef disabling multiple rules at once require additional newline between comments. 

```go
// disable:revive:rule1
/* there should be newline, otherwise rule2 will be still enabled*/
// disable:revive:rule2
```

**To Reproduce**
1. I updated revive `go get -u github.com/mgechev/revive`
2. Can be tested with docker only in isolated environment with default `config.toml`:

```shell
echo '
        FROM golang:alpine
        RUN apk add --no-cache git
        RUN go get -u -v github.com/mgechev/revive
        RUN mkdir -p /go/src/test
        RUN echo "package main" > /go/src/test/main.go && \
                echo "" >> /go/src/test/main.go && \
                echo "// revive:disable:var-naming" >> /go/src/test/main.go && \
                echo "// revive:disable:increment-decrement" >> /go/src/test/main.go && \
                echo "func main() {" >> /go/src/test/main.go && \
                echo "  var invalid_name = 0" >> /go/src/test/main.go && \
                echo "  invalid_name += 1" >> /go/src/test/main.go && \
                echo "  println(invalid_name)" >> /go/src/test/main.go && \
                echo "}" >> /go/src/test/main.go
        RUN revive test' | docker build --no-cache -
```

3. Also can be tested on local machine for this file:

```go
package main

// revive:disable:var-naming
// revive:disable:increment-decrement
func main() {
  var invalid_name = 0
  invalid_name += 1
  println(invalid_name)
}
```

```shell
revive ...
```

**Expected behavior**
Revive should exit with 0 RC and with empty output to STDERR and STDOUT. 

**Logs**
```
main.go:7:3: should replace invalid_name += 1 with invalid_name++
```

**Desktop (please complete the following information):**
 - OS: 
   * darwin
   * linux
 - Version of Go: 
   * go version go1.11.4 darwin/amd64
   * go version go1.11.4 linux/amd64

**Additional context**
Probably caused by new regexp: https://github.com/mgechev/revive/blob/c878d3090df36b2b57f9427ec7ea29be528e19ef/lint/file.go#L130

**Comment by chavacava** [comment]
Hi @mrVanboy, thanks for reporting the bug.
I've reproduced it.
As you suggest, the problem is caused by the new regexp that does not expect two revive directives in the same line (GO parser returns a single line for all contiguous comment lines)
I will try to fix the bug ASAP, meanwhile, to get the expected behavior you can write

```go
// revive:disable:var-naming,increment-decrement
func main() {
	var invalid_name = 0
	invalid_name += 1
	println(invalid_name)
}
```
